### DISCUSSION TITLE
Default values

### DISCUSSION THREAD

**Discussion by nhorton** [jnunemaker/flipper#798] [General]
We currently have code to set the config to "enabled" for some features in our staging environment by default to force those to be seen and tested. But we have tons of trouble with this code for various reasons. At its core, it is working around that we really need a way to set the default value for a given gate in an initializer or such. It would ideally only be used if there is no persisted gate in the DB. Any interest in supporting this, or good reference code on doing it ourselves?

**Comment by jnunemaker** [comment]
Hi @nhorton! 

There has been some good discussion on this before in https://github.com/flippercloud/flipper/issues/482.

I will say I no longer have a need for it thanks to Cloud. Cloud supports adding Staging as another environment (or whatever else you need). You can then override production however you want so it works differently. It's slick. Happy to show it to you sometime or you can sign up for the free tier and try it. Multi-env is included in Cloud free.
